common/org.testeditor.dsl.common.testing/pom.xml

```diff
@@ -13,6 +13,13 @@
 
 	<artifactId>org.testeditor.dsl.common.testing</artifactId>
 	<packaging>eclipse-plugin</packaging>
+	<dependencies>
```

Comment:
We should have a line before and after the `<dependencies>` for better readability